clients/Barracks.spec.js

```diff
@@ -36,7 +37,7 @@ describe('Barracks', () => {
       name: 'test update',
       versionId: '2.3',
       uuid: '123456789',
-      packageInfo: { id: '123456' }
```

Comment:
Please, check your text editor syntax configuration, not a huge fan of the space removed in inline objects (and the huge diff that comes with it)